symfony/phpunit-bridge/4.3/bin/phpunit

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env php
+<?php
+
+if (!file_exists(dirname(__DIR__).'/vendor/symfony/phpunit-bridge/bin/simple-phpunit.php')) {
+    echo "Unable to find the `simple-phpunit` script in `vendor/symfony/phpunit-bridge/bin/`.\n";
+    exit(1);
+}
+
+if (false === getenv('SYMFONY_PHPUNIT_VERSION')) {
+    putenv('SYMFONY_PHPUNIT_VERSION=6.5');
+}
+if (false === getenv('SYMFONY_PHPUNIT_REMOVE')) {
+    putenv('SYMFONY_PHPUNIT_REMOVE=');
+}
+if (false === getenv('SYMFONY_PHPUNIT_DIR')) {
+    putenv('SYMFONY_PHPUNIT_DIR='.__DIR__.'/.phpunit');
+}
+
+require dirname(__DIR__).'/vendor/symfony/phpunit-bridge/bin/simple-phpunit.php';
```

Comment:
instead of creating a new version of the recipe, what about doing a file_exists check here in the 4.1 recipe?

Reply:
I can do so  to but don't think it's needed.

There is no BC break in Bridge/PhpUnit 4.3 so the check is only decreases number of recipe versions but make the code less cleaner. I prefer cleaner code :)

Reply:
From a maintenance perspective it would make our life easier :)

Reply:
Fair enough :)

Updated 4.1 recipe

Reply:
Looking at the resulting code, I decide to follow your initial proposal and add a recipe for 4.3.
I created symlinks to 4.1 files.

Reply:
Will flex actually work fine with simlinks? :smile:

Reply:
It does already ;)